2022/02/09-telecon/README.md

```diff
@@ -66,9 +66,1112 @@ The Private Advertising Technology Community Group's meeting will be meeting two
+
+**Aram**:yes. In parallel
+
+**Brian**:Seems that lots of people are working in parallel directions. We should adopt something we like and find a way of working together. What’s our cadence
+
+**Aram**:will come back to that
+
+**Brad**:To clarify “adoption” - as per the charter: something the group wants to discuss, not an endorsement. A thing that is to be discussed with IPR protections with at least one other interested party.
+
+**EKR:** This CG can attempt to progress an idea towards adoption. Don't care what the word is. But we need a word for “work item for the group”, not just a thing being talked about. That’s an important distinction, and we need to be clear about it. Words should be clear.
+
+**Sean**:OUtput of a CG is a report.
+
+**EKR_: _**Basis of a report requires consensus
+
+**Aram**:Adopted can mean having a home for discussion, not recommended
+
+[EKR: This is not at the mic, but the text of the charter is quite clear that adoption means that there is consensus “that the work should be taken on and that the document is a good basis for a Community Group Report”, so it’s a higher bar than “there is agreement that this deserves some airtime so it gets IPR protection].
+
+
+<table>
+  <tr>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>
+   </td>
+  </tr>
+</table>
+
+
+Group Report.
+
+**..**:Next meeting - we’ll open an issue with polling. Back and forth on whether this should be monthly or every two months. Wondering if, given where we are, we should bias to monthly. Is there an objection to every month?
+
+**Martin:** 6-8 weeks
+
+**Bmay:** while there’s stuff to talk about, we should push. CGs can take a long time to get stuff done.
+
+**Aram**:6 weeks… any objections?
+
+**Sean**:Suggestion to splitting by a day. Will put the doodle poll up. Promise we’ll move the times around
+
+**Erik**: assumption is that we’ll put issues in GH, and have meaningful discussions there. Imagine that there are some small groups who might benefit from breakout live sessions. Are we okay with that? Discouraging? No position?
+
+**Aram**:same as privacy CG. There is a label. We’ll set up an issue template for an ad hoc meeting, can be discussed in that thread.
+
+**Bmay**:Concern - how to know that they're happening
+
+**Aram**:issues will email the list. When theres’ an ad hoc meeting, lets push a pull request to teh meetings repo
+
+**Erik**:sounds fine as a place to start
+
+**Aram**:that’s satisfactory. Sean to close
+
+**Sean**:thanks again. Looking forward to progress. Closed. Bye. Thanks all.
+
+
+#### 5m break
+
+
+#### Scheduling next meeting and discussing focus {#scheduling-next-meeting-and-discussing-focus}
+
+
+
+* Potential issues to pick up:
+    * [https://github.com/patcg/meetings/issues/19](https://github.com/patcg/meetings/issues/19)
+
+
+
+### Participants Day 2
```

Comment:
EKR was on day 2 as well - think his name ended up duplicated as 89 and 44 below in day 1